api/basic/get-pin-router.ts

```diff
@@ -30,6 +30,8 @@ export async function handler (event: APIGatewayProxyEventV2, context: Context):
 
   logger.info('Get pin request')
 
+  await delay(40000)
```

Comment:
What's this for?

Reply:
Timeout to validate apigw behavior, gw has a hard limit of 30s.
This pr it's for test purpose only

Reply:
ok, clear.